app/controllers/assets_controller.rb

```diff
@@ -2,27 +2,13 @@ class AssetsController < ApplicationController # rubocop:todo Style/Documentatio
   # WARNING! This filter bypasses security mechanisms in rails 4 and mimics rails 2 behviour.
   # It should be removed wherever possible and the correct Strong  Parameter options applied in its place.
```

Comment:
Lots of deletions in this controller, as most of these endpoints are replaced by those on either Labware or receptacle. In most cases they were giving a 500 if you attempted to access them. So I figured why update something that doesn't work.